### PR TITLE
give full efeatures & protocols examples in FCC docstring

### PR DESCRIPTION
**Pull request by AurelienJaquier**
@darshanmandge wanted to comment on PR #58 to use a full example in docstring, so I implemented it here

### CODE REVIEW

File: bluepyemodel/evaluation/fitness_calculator_configuration.py [hunk 1]
```diff
@@ -105,24 +105,55 @@ def __init__(
                 .. code-block::
 
                     [
-                        {"efel_feature_name": str, "protocol_name": str, "recording_name": str,
-                        "mean": float, "std": float, "efel_settings": dict}
+                        {
+                            "efel_feature_name": "Spikecount",
```
Comment (darshanmandge) [inline bluepyemodel/evaluation/fitness_calculator_configuration.py:109]: Are the keys of this dictionary and that of protocols described anywhere else?

File: bluepyemodel/evaluation/fitness_calculator_configuration.py [hunk 2]
```diff
@@ -105,24 +105,55 @@ def __init__(
...
+                            "protocol_name": "IDrest_130",
+                            "recording_name": "soma.v",
```
Comment (ilkilic) [inline bluepyemodel/evaluation/fitness_calculator_configuration.py:111]: Did you forget to include the protocol_name here? In the previous comment, you mentioned that "here, recording_name should have the format ``f"{protocol_name}.{location}.v"``,
                e.g. ``"IV_0.soma.v"``"

File: bluepyemodel/evaluation/fitness_calculator_configuration.py (hunk 1 above)
Comment (AurelienJaquier) [reply bluepyemodel/evaluation/fitness_calculator_configuration.py:109]: They are basically the arguments of `bpem.evaluation.efeature_configuration.EFeatureConfiguration` and `bpem.evaluation.protocol_configuration.ProtocolConfiguration`, so they are also described in these classes docstrings, respectively.

File: bluepyemodel/evaluation/fitness_calculator_configuration.py (hunk 2 above)
Comment (AurelienJaquier) [reply bluepyemodel/evaluation/fitness_calculator_configuration.py:111]: I mixed up the recording name which had the protocol name, and the one that did not in the previous PR ^^'

File: bluepyemodel/evaluation/fitness_calculator_configuration.py [hunk 3]
```diff
@@ -105,24 +105,55 @@ def __init__(
...
+                                "thresh_perc": 130.56103157894464,
+                                "duration": 2053.050884955752,
+                                "totduration": 3000.0,
+                                "holding_current": -0.13307330411636328
+                                }
+                            ],
+                            "recordings_from_config": [
+                                {
+                                "type": "CompRecording",
+                                "name": "IDrest_130.soma.v",
```
Comment (ilkilic) [inline bluepyemodel/evaluation/fitness_calculator_configuration.py:146]: and here shouldn't it be without the protocol_name?

File: bluepyemodel/evaluation/fitness_calculator_configuration.py (hunk 1 above)
Comment (darshanmandge) [reply bluepyemodel/evaluation/fitness_calculator_configuration.py:109]: Could you mention this here? Users might want to know the details what kind of values can be put in for each key. And if they are not described there then we should describe them there. Thanks!

**Review by ilkilic — APPROVED** [review]

File: bluepyemodel/evaluation/fitness_calculator_configuration.py (hunk 1 above)
Comment (darshanmandge) [reply bluepyemodel/evaluation/fitness_calculator_configuration.py:111]: Thanks!

**Review by darshanmandge — APPROVED** [review]

**Pull request merged**